addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoHandlerFactory.java

```diff
@@ -63,7 +69,8 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     protected ThingHandler createHandler(Thing thing) {
         ThingTypeUID thingTypeUID = thing.getThingTypeUID();
         if (thingTypeUID.equals(APIBRIDGE_THING_TYPE)) {
-            bridgeHandler = new NetatmoBridgeHandler((Bridge) thing);
+            WelcomeWebHookServlet servlet = registerWebHookServlet(thing.getUID());
```

Comment:
The `registerWebHookServlet()` method could return `null`. Should it be handled in some way?

Reply:
In fact, this case is handled by the bridge handler. The handler is ignoring the web hook servlet in case its registration was not done.

Reply:
Alright. I see. Hidden in the `getWebHookURI()` method.